client/src/connection/com/index.ts

```diff
@@ -114,12 +114,17 @@ export class COMSession extends Session {
 
       //write ODS output to work so that the session cleans up after itself
       const codeWithODSPath = code.replace(
-        "ods html5;",
-        `ods html5 path="${this._workDirectory}";`,
+        /\bods html5\b([^;]*;)/i,
+        `ods html5(id=_vscode_sas) path="${this._workDirectory}"$1`,
```

Comment:
I guess it would be good to wrap the id in the `wrapCodeWithOutputHtml` function.
It's different to the `path=` which specific to IOM. The test case for id also applies to Viya.

Reply:
Now make the change of id to `wrapCodeWithOutputHtml`